desc/objectives/linear_objectives.py

```diff
@@ -1376,6 +1363,151 @@ def compute(self, Z_lmn, **kwargs):
         return fixed_params
 
 
+class FixModeLambda(_FixedObjective):
+    """Fixes Fourier-Zernike lambda coefficients.
+
+    Parameters
+    ----------
+    eq : Equilibrium, optional
+        Equilibrium that will be optimized to satisfy the Objective.
+    target : float, ndarray, optional
+        Fourier-Zernike lambda coefficient target values. If None,
+         uses Equilibrium's lambda coefficients.
+    bounds : tuple, optional
+        Lower and upper bounds on the objective. Overrides target.
+        len(bounds[0]) and len(bounds[1]) must be equal to Objective.dim_f
+    weight : float, ndarray, optional
+        Weighting to apply to the Objective, relative to other Objectives.
+        len(weight) must be equal to Objective.dim_f
+    normalize : bool
+        Whether to compute the error in physical units or non-dimensionalize.
+    normalize_target : bool
+        Whether target should be normalized before comparing to computed values.
+        if `normalize` is `True` and the target is in physical units, this should also
+        be set to True.
+    modes : ndarray, optional
+        Basis modes numbers [l,m,n] of Fourier-Zernike modes to fix.
+        len(target) = len(weight) = len(modes).
+        If True uses all of the Equilibrium's modes.
+        Must be either True or specified as an array
+    name : str
+        Name of the objective function.
+
+    """
+
+    _scalar = False
+    _linear = True
```

Comment:
since you're inheriting from `_FixedObjective` i don't think these are needed but you do need to specify `target_arg`